src/client/components/editCropImage/editCrop.component.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './editCrop.style.css';
+import PropTypes from 'prop-types';
+import penImage from '../../assets/images/edit-icon.png';
+
+export default function Image({ srcPath, altText }) {
+  return (
+    <div>
+      <img className="edit-image" src={srcPath} alt={altText} />
+      <button type="submit" className="edit-icon">
```

Comment:
This className could be `edit-btn` instead of `edit-icon` and then the image inside could have a className `edit-icon`, to be more accurate.

Reply:
updated